### DISCUSSION TITLE
How to remove collection id prefix "pbc_"?

### DISCUSSION THREAD

**Discussion by boldeuler** [pocketbase/pocketbase#6024] [Q&A]
Starting of v23, creating a collection via the api adds "pbc_" prefix and only numbers, example: "pbc_3640472499".

How to make it generate 0-9 a-z{15} same to recrods?

I know that I can set the id in the post request, is it possible to define it in pocketbase directly?

Thanks for the great job!

**Comment by ganigeorgiev** [comment]
Please elaborate what you are trying to do and why the currently autogenerated id is a problem in the first place (the numbers are crc32 checksum of the collection type+name).

You can set your own id by modifying the `e.Collection.Id` field inside [`OnCollectionCreate`](https://pocketbase.io/docs/go-event-hooks/#oncollectioncreate) or [`OnCollectionCreateRequest`](https://pocketbase.io/docs/go-event-hooks/#oncollectioncreaterequest) hook but I don't recommend replacing them with a random value because the entire idea behind the new ids was to prevent eventual migration conflicts when merging collections created/modified by different environments.
